modules/observe/adoption-insights/proc-configure-adoption-insights.adoc

```diff
@@ -2,24 +2,9 @@
 [id="proc-configure-adoption-insights_{context}"]
 = Configuring the Adoption Insights plugin in {product}
 
-You can enable the Adoption Insights plugin by configuring the {product} Helm chart or the {product} Operator ConfigMap.
+The Adoption Insights plugin is preinstalled and enabled on a {product-short} instance by default. You can disable or enable the Adoption Insights plugin, and change other parameters, by configuring the {product} Helm chart or the {product} Operator ConfigMap.
 
-.Procedure
-
-* To configure the Adoption Insights plugin in {product-short}, in your {product} `app-config.yaml` file, add the following code:
```

Comment:
We do need this now as well for users to be able to configure certain parameters.